compiler/next/lib/parsing/bison-chpl-lib.h

```diff
@@ -1,4 +1,4 @@
-/* A Bison parser, made by GNU Bison 3.8.2.  */
+/* A Bison parser, made by GNU Bison 3.7.6.  */
```

Comment:
We have been trying to keep to the same bison version to minimize these diffs. AFAIK the Homebrew bison is 3.8.2.